### ISSUE TITLE
Sentry.setVersion not working (maybe I'm calling in the wrong place?)

### ISSUE THREAD

**Issue by ndbroadbent** [getsentry/react-native-sentry#106]
I'm using `react-native-sentry` version `0.12.11`.

In my `index.android.js`, I have the following code:

```js
import { AppRegistry } from 'react-native'
import CodePush from 'react-native-code-push'
import { Sentry } from 'react-native-sentry'
import Config from 'config'
import App from './src'

if (!__DEV__) {
  Sentry.config(Config.Sentry.DSN).install()
  CodePush.getUpdateMetadata().then((update) => {
    if (update) {
      const sentryVersion = `codepush:${update.label}`
      console.log(`Got CodePush update. Setting Sentry version to: "${sentryVersion}"`)
      Sentry.setVersion(sentryVersion)
    }
  })
}

AppRegistry.registerComponent('sudoblock', () => App)
```

In my crash report, I see the following 'bread crumb':

```
Got CodePush update. Setting Sentry version to: "codepush:v20"
```

However, the "release" in the crash report is still: `com.ndbroadbent.sudoblock-1.0.5`. I think it should be  `com.ndbroadbent.sudoblock-codepush:v20`.

I'm also seeing `Version	1.0.5` under the "APP" section in the Sentry UI.

Is this because I am calling `Sentry.setVersion` too early? Or am I doing something wrong?

**Comment by ndbroadbent** [comment]
[Looking at the code](https://github.com/getsentry/react-native-sentry/blob/e21ca721293d419840f84fc96fbaf232341df19d/lib/Sentry.js#L157-L168), it seems that `setVersion` doesn't do anything to update the release. And looking at the JSON of my crash report, the "version" key isn't even sent. I will try using `setRelease` instead of `setVersion`.

I would really appreciate an update to [the CodePush docs](https://docs.sentry.io/clients/react-native/codepush/) to make this clearer.

**Comment by ndbroadbent** [comment]
For some reason, `setRelease` isn't working either.

My new code in `index.android.js`:

```js
import { AppRegistry } from 'react-native'
import CodePush from 'react-native-code-push'
import { Sentry } from 'react-native-sentry'
import Config from 'config'
import App from './src'

if (!__DEV__) {
  Sentry.config(Config.Sentry.DSN).install()
  CodePush.getUpdateMetadata().then((update) => {
    if (update) {
      const sentryRelease = `com.ndbroadbent.sudoblock-codepush:${update.label}`
      console.log(`Got CodePush update. Setting Sentry release to: "${sentryRelease}"`)
      Sentry.setRelease(sentryRelease)
    }
  })
}

AppRegistry.registerComponent('sudoblock', () => App)
```

Breadcrumb from crash report:

```
Got CodePush update. Setting Sentry release to: "com.ndbroadbent.sudoblock-codepush:v21"
```

JSON of error event:

```
...
id: "43e7f4d72b584519870d70022e22ba5a",
project: 173101,
release: "com.ndbroadbent.sudoblock-1.0.5",
...
```

I need some help figuring out why the `release` is not being set properly.

**Comment by HazAT** [comment]
Hey @ndbroadbent `setVersion` should definitely be the right choice. Also, your implementation looks fine. Can you please upgrade to the latest `0.13.x` and check again. 
I've just tried it locally and it worked.
<img width="439" alt="screen shot 2017-06-26 at 12 18 05" src="https://user-images.githubusercontent.com/363802/27535075-8c6fed50-5a69-11e7-92bc-93072175fa32.png">

**Comment by f0rr0** [comment]
@HazAT I use this script: https://github.com/getsentry/react-native-sentry/issues/79#issuecomment-308501186 to release on codepush. This creates a release on sentry. However, using setVersion creates another release with the same name (for eg: codepush:v26). This second release does not have my source maps etc.

**Comment by HazAT** [comment]
Hey, I've just released `0.15.0` which has tests for this issue. So it is working, do you still have problems with it?
